.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "cds-hooks"]
+	path = src/rems-cds-hooks
+	url = https://github.com/mcode/rems-cds-hooks.git
+	branch = prefetch-submodule
```

Comment:
The rems-cds-hooks code has been merged into dev. We can update this branch to dev now.